src/rendering/vcFenceRenderer.cpp

```diff
@@ -341,9 +258,11 @@ udResult vcFenceRenderer_CreateSegmentVertexData(vcFenceRenderer *pFenceRenderer
   // start segment
   current = udFloat3::create(pSegment->pCachedPoints[0]);
   next = udFloat3::create(pSegment->pCachedPoints[1]);
-  expandVector = vcFenceRenderer_CreateStartJointExpandVector(current, next, pFenceRenderer->config.ribbonWidth, worldUp);
-  pVerts[vertIndex + 0].ribbonInfo = udFloat4::create(expandVector, 0.0f);
-  pVerts[vertIndex + 1].ribbonInfo = udFloat4::create(-expandVector, 1.0f);
+
+  //We don't use the full vec4, but leave it as is; we may want to use it laster to support
+  //individual extend vectors for each point in the line.
+  pVerts[vertIndex + 0].ribbonInfo = udFloat4::create(0.0, 0.0, 0.0, 0.0f);
+  pVerts[vertIndex + 1].ribbonInfo = udFloat4::create(0.0, 0.0, 0.0, 1.0f);
```

Comment:
Removing the joint expansion (even if it is slightly broken) seems like a massive step backwards?

Reply:
It didn't look like the vertical fence used the expand vector, and I thought we could get away with a similar idea for the horizontal fence. Long story short I've just come back to this PR and I agree we shouldn't the joint expansion.